forum/src/main/java/org/exoplatform/platform/qa/ui/forum/pageobject/ForumPermission.java

```diff
@@ -170,4 +182,37 @@ public void selectPermGroupMemberMes(String groupPath, String member) {
     evt.click(ELEMENT_MESSAGE_SELECT_RIGHT_PARENT_GROUP.replace("$group", member), 0, true);
     evt.waitForElementNotPresent(ELEMENT_MESSAGE_SELECT_MEMBERSHIP_POPUP);
   }
+  public void addUserPermission(String user, String category, String forum, String topic)
+  {
+    if(!topic.isEmpty())
+    {
```

Comment:
Brackets should be at the end of line. Please use eXo formatting files : http://developer.exoplatform.org/#id-ide-settings-to-work-on-exo-platform-projects